### ISSUE TITLE
Meridian convergence looks good but seems to be applied in reverse

### ISSUE THREAD

**Issue by nosy-b** [iTowns/itowns#1215]
Meridian convergence looks good but seems to be applied in reverse ( [Pull #1212  ](https://github.com/iTowns/itowns/pull/1212))

Weird effect when using:
`var quat_crs2crs = itowns.OrientationUtils.quaternionFromCRSToCRS("EPSG:2154", "EPSG:4978")(coords);`

What worked to convert a quaternion from local (L93) to WGS 84:
```
var quaternion = new THREE.Quaternion(x,y,z,w)  // Local
var coords = new itowns.Coordinates('EPSG:4978', pos.x, pos.y, pos.z);
quaternion.premultiply(itowns.OrientationUtils.quaternionToCRS('EPSG:2154')(coords));
quaternion.premultiply(itowns.OrientationUtils.quaternionToCRS('EPSG:4978')(coords));

```
@mbredif Could you check? :)

**Comment by mbredif** [comment]
I confirm the bug, the meridian convergence has the right magnitude but is applied in the wrong direction. I'll push a fix soon. Thanks for prooftesting it with real data ;)

**Comment by zarov** [comment]
If you also have a unit test for this case, please consider adding it as well ;)

**Comment by mbredif** [comment]
the unit test is already there. it passed because it had the same confusion "from<->to".

**Comment by gchoqueux** [comment]
this will be fixed in the PR #1218
